### PR TITLE
62 feature editUserData Hubach

### PR DESCRIPTION
**Pull request by innomaxx**

### CODE REVIEW

**Review by andrii-kondratiuk — APPROVED** [review]

File: controllers/user-controller.js [hunk 1]
```diff
@@ -11,12 +11,39 @@ exports.getAllUserData = async (req, res) => {
 
     const getUser = await userModel.findOne({ where: { id: userID } });
     if (!getUser) return res.status(500).send({ message: 'Can`t find user with this id' });
-    
+
     const userData = getUser.dataValues;
+    console.log(userData);
```
Comment (Dmitry528) [inline controllers/user-controller.js]: Its all okay, but delete console.log

**Review by Dmitry528 — COMMENTED** [review]
Delete console.log

File: controllers/user-controller.js (hunk 1 above)
Comment (innomaxx) [reply controllers/user-controller.js]: done

**Review by Dmitry528 — APPROVED** [review]
Sign in like user@email.com, and change name, surname, nickname and after redirect to view profile page, i have seen this changes, good work

**Review by marrybbb — APPROVED** [review]

**Pull request merged**